circle.yml

```diff
@@ -294,7 +320,7 @@ jobs:
       - deploy:
           name: Publish to Maven
           command: |
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then make run-android-upload-archives ; fi
+            if [ "${CIRCLE_BRANCH}" == "release-agua" ]; then make run-android-upload-archives ; fi
```

Comment:
@tobrun shouldn't be `master`?

Reply:
we continue to work from the agua release branch until that 5.2.0 is finalised, this configuration is correct for that. Not having this config on master is an issue as we have been building snapshots from master and release-agua..